.github/workflows/test.yml

```diff
@@ -11,14 +11,14 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-12, ubuntu-22.04, ubuntu-20.04]
+        os: [macos-14, ubuntu-22.04, ubuntu-24.04]
         glpk: ['4.65', '5.0']
-        python-version: ['3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.9', '3.10', '3.11', '3.12']
```

Comment:
Python 3.8 may be EOL upstream but its still the version included in Ubuntu 20.04. I'm tweaking the build matrix in #48 to get coverage on the relevant combinations while minimizing the total number of builds.